### PR TITLE
[UserNotifications] Update UserNotifications to Xcode 8 beta 3

### PR DESCRIPTION
**Pull request by dalexsoto**

### CODE REVIEW

**Comment by xamarin-release-manager** [comment]
Build failure

File: src/usernotifications.cs [hunk 1]
```diff
@@ -28,12 +28,14 @@ namespace XamCore.UserNotifications {
 	[Native]
 	public enum UNErrorCode : nint {
 		NotificationsNotAllowed = 1,
-		AttachmentInvalidUrl = 100,
+		AttachmentInvalidURL = 100,
```
Comment (spouliot) [inline src/usernotifications.cs]: nope, please use `Url`

**Comment by spouliot** [comment]
It's fine to merge once URL typo is fixed (and bots intro tests pass).

File: src/usernotifications.cs (hunk 1 above)
Comment (dalexsoto) [inline src/usernotifications.cs]: Ouch good catch, Copy/Paste mistake

**Pull request merged**